ili9341/registers.go

```diff
@@ -0,0 +1,137 @@
+package ili9341
+
+type Rotation uint8
+
+const (
+
+	// register definitions based on source:
+	// https://github.com/adafruit/Adafruit_ILI9341/blob/master/Adafruit_ILI9341.h
```

Comment:
Since this is a reinterpretatation but not the same code, I think you can leave these first 2 lines lines but remove the rest of the comments, since this is not just copying that source code.

As far as which displays this code works with, that should probably be in GoDoc readable format somewhere like in the beginning of `ili9341.go`